frontend/pages/create-project.tsx

```diff
@@ -63,99 +64,116 @@ const CreateProject: React.FC = () => {
 
     const formik = useFormik({
         initialValues: {
-            projectName: '',
-            aboutProject: '',
-            selectedFile: null,
-            listOfParticipant: [] as string[],
-            newList: '',
+            name: '',
+            description: '',
+            selectedFile: '',
+            members: [] as string[],
+            newMembers: '',
             material: [] as string[],
-            newMaterial: '',
+            newMaterials: '',
```

Comment:
Why's the use for this property ? I don't think a project will add new material to the HUB

Reply:
it's the list of materials whose will be used in the project, i will clarify the name

Reply:
still è.é